android/app/src/main/java/org/fossasia/openevent/utils/AuthUtil.java

```diff
@@ -34,23 +37,22 @@
 
     private static Authenticator authenticator;
 
-    public static void signUpUser(Context context, String email, String password, ProgressBar progressBar) {
-        APIClient.getOpenEventAPI().signUp(User.builder().email(email).password(password).build())
+    public static void loginUserWithoutProgressBar(String email, String password,
+                                                   MutableLiveData<Pair<Integer,String>> loginResponse,
+                                                   CompositeDisposable compositeDisposable) {
+        compositeDisposable.add(APIClient.getOpenEventAPI().login(new Login(email, password))
```

Comment:
For now, move it into ViewModel because it is simply taking the environment of ViewModel and passing into the arguments. We'll think of something better when there is time for sharing code